src/components/Section/index.tsx

```diff
@@ -78,6 +79,10 @@ type SectionProps = ChildrenProps & {
     renderSubtitle?: () => ReactNode;
 };
 
+function isIllustrationLottieAnimation(illustration: DotLottieAnimation | IconAsset | undefined): illustration is DotLottieAnimation {
+    return typeof illustration === 'object';
```

Comment:
Hm, are we 100% sure that IconAsset will never return an object? We can consider using [`instanceof` operator](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#instanceof-narrowing).